docs/src/main/paradox/scala/http/routing-dsl/directives/file-and-resource-directives/index.md

```diff
@@ -1,7 +1,7 @@
 # FileAndResourceDirectives
 
 Like the @ref[RouteDirectives](../route-directives/index.md) the @unidoc[FileAndResourceDirectives] are somewhat special in akka-http's routing DSL.
-Contrary to all other directives they do not produce instances of type `Directive[L <: HList]` but rather "plain"
+Contrary to all other directives they do not produce instances of type @scala[`Directive[L <: HList]`]@java[`Directive<L extends HList>`] but rather "plain"
```

Comment:
This is just a translation to Java Syntax

Reply:
Hmm not sure if that helps Java users :)

Maybe we can cleanup the whole paragraph a bit and say

```
Like the @ref[RouteDirectives](../route-directives/index.md) the @unidoc[FileAndResourceDirectives] are somewhat
special in akka-http's routing DSL. Most other directives wrap an inner route and are therefore used as inner nodes of
the route tree. These directives, instead, are indeed instance of `Route`, i.e.  **leaves** of the route tree that handle a
request themselves without passing it on to an inner route.
```

WDYT?

Reply:
Agree, and I think it didn't really help a subgroup of Scala users either. I like your phrasing, fix is on the way